cmd/kops/validate_cluster.go

```diff
@@ -180,6 +207,48 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 
 }
 
+// getGroupMessages returns the cloud group messages
+func getGroupMessages(validationFailed error, validationCluster *validation.ValidationCluster, cluster *api.Cluster) []cloudinstances.CloudInstanceGroupMemberMessage {
+	if validationFailed != nil {
+		m, err := validationCluster.CollectCloudGroupMessages(cluster)
+		if err != nil {
+			// eating err because some of the cloud providers do not have this implemented
+			glog.Infof("unable to get cloud  group messages %v.", err)
+			return nil
+		}
+
+		return m
+	}
+	return nil
+}
+
+func validateClusterOutputTableGroupMessages(validationCluster *validation.ValidationCluster, out io.Writer) error {
+	if len(validationCluster.GroupMessages) != 0 {
+		groupTable := &tables.Table{}
+		groupTable.AddColumn("GROUP", func(s cloudinstances.CloudInstanceGroupMemberMessage) string {
+			return s.HumanName
+		})
+
+		groupTable.AddColumn("MESSAGE", func(s cloudinstances.CloudInstanceGroupMemberMessage) string {
+			return s.Message
+		})
+
+		groupTable.AddColumn("STATUS", func(s cloudinstances.CloudInstanceGroupMemberMessage) string {
+			return s.StatusCode
+		})
+
+		fmt.Fprintln(out, "\nCLOUD GROUP MESSAGES")
+		err := groupTable.Render(validationCluster.GroupMessages, out, "GROUP", "MESSAGE", "STATUS")
```

Comment:
Are there cases where this could return a large number of messages?  Maybe we should set a default limit to the number of messages and allow an override on the CLI (or that would be an easy add on PR)?  

Running this on our prod cluster I get:
```
bazel-bin/cmd/kops/darwin_amd64_stripped/kops validate cluster | wc

881    5948   77252
```

Reply:
How many messages are on your ASG's?  I do not have a large cluster to test this on.  Limiting this to the last 5 messages or so would not be a bad thing.

Reply:
Should we have an option to turn it off??

Reply:
@chrislovecnm Looking at one of our ASGs, we have 600+ messages, basically just autoscaling and such. 5-10 seems fine to me, enough to point out an issue likely. I think an option to turn off could come later and would not be necessary up front but that's just me.